composer.json

```diff
@@ -124,6 +124,9 @@
             "drupal/socialbase": {
                 "Revert: As a user I should receive an email confirmation when I enroll in an event": "https://www.drupal.org/files/issues/2022-05-03/socialbase-revert_accidentally_merged_and_released_changes_when_distro_pr_was_still_open-3273222-8.patch"
             },
+            "drupal/socialblue": {
+                "Add primary color on hover to the user name on the profile preview modal": "https://www.drupal.org/files/issues/2022-05-03/socialblue-previewer-username-hover-color-3278142-3.patch"
```

Comment:
@OH17L You can remove the patch from PR since we will merge theme changes with this PR at the same time and this all will be in the new release theme/distro.

Reply:
I'd prefer to use the default workflow when patches will be deleted only after switching the theme to a newer version which contains these patches.